docs/examples/django/client.py

```diff
@@ -0,0 +1,45 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
I'm wondering if these examples should go under `http` or should we move `flask` examples out into their own folder as well?

Reply:
Sorry, what do you mean with _move `flask` examples out into their own folder as well_? There is no mention to Flask in any part of the Django examples. Also, this file uses `requests` only as a client, to make the Django server respond with a Span, but this is not a `requests` instrumentation example. I can remove the mention to the client output from this example if that makes this more clear.

Reply:
Ahh right my mistake. I thought the `server.py` example was specific to `flask`.